store/timer_layer.go

```diff
@@ -7317,7 +7296,6 @@ func (s *TimerLayerWebhookStore) ClearCaches() {
 		}
 		s.Root.Metrics.ObserveStoreMethodDuration("WebhookStore.ClearCaches", success, elapsed)
 	}
-	return
```

Comment:
This file is auto-generated, so you should be modifying the generator, and regenerated again.